portfolio/src/main/webapp/style.css

```diff
@@ -4,7 +4,7 @@
   width: 650px;
 }
 
-#greeting-container {
+#quote-container {
```

Comment:
This should be "song-container" now right?

Reply:
Yes, it should. Thank you.